.github/workflows/build.yml

```diff
@@ -28,6 +28,10 @@ jobs:
             -DCMAKE_BUILD_TYPE=${{matrix.build_type}} -DPAHO_WITH_SSL=1 -DBUILD_DEB_PACKAGE:BOOL=1 -DDEB_PACKAGE_LIBCPP_VERSION=$LIBCPP_VERSION
           cmake --build . -j
           make package
+      - name: Run Tests
+        run: |
+          cd build/Dashboard-Client-build
+          ctest -V
```

Comment:
Why is `-C ${{matrix.build_type}}` missing here?

Reply:
Good point. I fixed this.

Reply:
> Why are three tests removed?
These tests are for the old Unified Automation SDK.